nilearn/tests/test_datasets.py

```diff
@@ -27,14 +27,22 @@
 file_mock = None
 
 
+def setup_silence():
+    # Wrap functions with versions that have verbose=0
+    datasets._get_dataset_dir = silence_fn(datasets._get_dataset_dir)
+    datasets._uncompress_file = silence_fn(datasets._uncompress_file)
+    datasets._fetch_file = silence_fn(datasets._fetch_file)
+    datasets._fetch_files = silence_fn(datasets._fetch_files)
+    datasets._chunk_read_ = silence_fn(datasets._chunk_read_)
```

Comment:
> Actually thinking about it a bit more, why is it not enough to pass verbose=0
> into the tested functions?

That would seem like the right thing to do to me. Unless the purpose is
to trigger the printing code in order to test it.